kalite/distributed/static/js/distributed/rating/views.js

```diff
@@ -160,13 +148,31 @@ var StarView = BaseView.extend({
     },
 
     render: function() {
-        this.$el.html(this.template(this.model.attributes));
+        var template_options = {};
+        _.extend(template_options, this.model.attributes, {title: this.title, lowest_value: this.label_values[1], highest_value: this.label_values[5]});
+        this.$el.html(this.template(template_options));
         this.rating_change();
     },
 
-    rate_value_callback: function(ev) {
-        var val = $(ev.target).attr("data-val");
+    // Throttled so it's triggered only once when several children are the targets of the "click" event
+    rate_value_callback: _.throttle(function(ev) {

```

Comment:
I'd suggest debounce instead of throttle (doesn't need to be saving constantly during the typing -- adds extra load -- just when they pause, as that's when they may then go on to close it). And could be 2 secs instead of 100ms.